Automattic/wordpress-importer
# Import from URL: Hook up network call

This hooks the URL entry form up to a network call to actually get the WXR and import it.

### To Test

• Apply D24540-code on your sandbox
• Sandbox the WPCOM REST API
• Run `master` branch of Jetpack (or the current stable release)
• Run this branch of `wordpress-importer`
• Build the UI `npm i; npm run build`
• Browse to `Tools | Import | WordPress Importer <Run Importer>`
• Enter a supported URL and complete the flow

Also test the File import flow -- it should "still work"
